src/qdisp/UberJob.h

```diff
@@ -68,7 +67,10 @@ class UberJob : public std::enable_shared_from_this<UberJob> {
     virtual void runUberJob();
 
     /// Kill this UberJob and unassign all Jobs so they can be used in a new UberJob if needed.
-    void killUberJob();
+    /// @return true if the UberJob results were stopped from merging. False means
+    ///         the results for this UberJob were already being merged or were merged before
+    ///         killUberJob was called.
+    bool killUberJob();
```

Comment:
The name of the method has redundancy:
```
class UberJob {
    bool killUberJob();
};
```
This adds extra impedance when reading/comprehending the code.
Perhaps, operations on the instance of the class could be described by simple verbs? Something along these lines:
```
class UberJob {
    virtual void run();
    bool kill();
    ...
};
```